frontend/src/views/SQR/Form.vue

```diff
@@ -61,12 +61,22 @@
                   flexWrap: 'wrap'
                 }"
               >
-                <v-btn v-if="!isCompleted" @click="saveDraft">
+                <v-btn v-if="!isLoadedFromCompletedBranch" @click="saveDraft">
                   Save draft
                 </v-btn>
-                <v-btn type="submit" color="primary" class=" mx-2"
-                  >Submit</v-btn
+                <v-btn
+                  v-if="
+                    $can('submit', {
+                      modelName: $subjects.SQR.form,
+                      isMarkedDone
+                    })
+                  "
```

Comment:
Add an alternative message for those who don't have access to Submit. Message: “Form is marked as done. Please contact the coordinator.”

Reply:
When the allotment is validated if they have no access to submit this is what's shown 
![image](https://user-images.githubusercontent.com/21045132/73324529-435ceb00-4219-11ea-824a-8148f769a7e0.png)

Reply:
Oh I see. Let's refactor it a bit (I'll add to the card as well).

If the form is marked as Done then:

- The form should still load.
- There should be a warning alert in the top “The submission is finalized.”
- All the controls should be read only and there should be no Submit button — for those who cannot update finalized submissions.